webview/src/setup/components/studio/CliIncompatible.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Feels like we're trying to head in the direction of splitting out components. LMK if I got this wrong.

Reply:
This is going to help greatly in keeping styles simple and easy to clean out. It'll also help with Codeclimate issues and even tests.